### DISCUSSION TITLE
Datetime interface "Set to now" should set seconds to 0 when "Include seconds" is disabled.

### DISCUSSION THREAD

**Discussion by moeriki** [directus/directus#7779] [Feature Requests]
Because editors are lazy they will often "Set to now" and then change the month/day/hour.

Now it's kind of strange to have the current seconds in the persisted datetime when there is no way for any editor to edit it.

**Comment by rijkvanzanten** [comment]
Makes total sense to me. Wanna open a PR for that?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
